src/database_router.js

```diff
@@ -0,0 +1,36 @@
+const router ={}
+const logger = require('../lib/log')(__filename)
+const dbModule = require('../sequelize/db')
+
+router.patch =  async(req,res)=>{
+    const {setDBPassword} = require('../lib/users')
```

Comment:
Why are you requiring inside of a function?

Reply:
at the top of the file:

```
const userModule = require(....)

router.patch = async (req, res) => {
  ...
  userModule.setDBPassword(....)
  ...
}
```

Reply:
solved and understand how mock function and constructor works!